src/Kernel.EFSupport/Kernel.EFSupport.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <Version>1.0.0</Version>
+        
+        <TargetFramework>net6.0</TargetFramework>
+        
+        <AssemblyName>LT.DigitalOffice.Kernel.EFSupport</AssemblyName>
+        <RootNamespace>LT.DigitalOffice.Kernel.EFSupport</RootNamespace>
+        <IsPackable>true</IsPackable>
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+        <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+
+        <GenerateDocumentationFile>false</GenerateDocumentationFile>
+
+        <OutputType>Library</OutputType>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.59" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.5" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="6.0.5" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.5" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.5" />
+        <PackageReference Include="Serilog.AspNetCore" Version="4.1.0" />
```

Comment:
why serilog here?

Reply:
for "Log" and "using Serilog" in "GetConnectionString"

Reply:
fixed with new Kernel version (1.1.61)